ngageoint/opensphere
# feat(heatmap): moved heatmap cesium sync to cesium

Moved implementation of HeatmapSynchronizer to Cesium as a simple workaround to handle the heatmap canvas when swapping back and forth between WebGLRenderer plugins

Testing Steps:
1. Opensphere | http://red-curtain.surge.sh/opensphere/
2. Add a Data Layer and load some data
3. Right click the Feature layer > Click "Generate Heatmap"
4. Verify a heatmap is drawn
5. Zoom in fairly close to the area
6. Verify the heatmap becomes more detailed as you get closer
(No further testing needed here; the change was simply moving the Synchronizer; not changing its functionality)
